[AC-1406] Request for quote use case documentation (#1373)
spryker/spryker-docs: docs/pbc/all/request-for-quote/use-cases-request-for-quote-processes.md

---
title: "Use cases: Request for Quote processes"

> description: This document describes the use cases for Requests for Quote.

Reviewer: Here I'd say its either Requests for Quotes or Request for Quote, the first if its a generic term, the second if its a name.

Author: Request for Quote is the name of the feature.

Reviewer: description: This document describes the use cases for Request for Quote.

then I think would be best

last_updated: Nov 15, 2022
template: concept-topic-template
---

> ## Use Case 1: Long waiting time until Request for Quote (RFQ) gets processed

Reviewer 2: ```suggestion
## Use case 1: Long waiting time until Request for Quote (RFQ) gets processed
```

> ## Use case 1: Long waiting time until Request for Quote (RFQ) gets processed

Author: ```suggestion
## Use case 1: Long waiting time until Request for Quote gets processed
```

> The [request for quote](/docs/pbc/all/request-for-quote/request-for-quote.html) process is currently not entirely efficient because it is manual. As a result, sometimes, customers have to wait for a long time until their RFQ gets approved or denied. 

Reviewer: ```suggestion
The [Request for Quote](/docs/pbc/all/request-for-quote/request-for-quote.html) process is currently not fully efficient because it is manual. As a result, sometimes customers have to wait a long time until their Request for Quote (RFQ) gets approved or denied. 
```
Same as above, if its a specific term then capitalised letters if not then no.

Author: I've adjusted your comment a bit.

### Solution

> To solve this issue, merchants must react faster to quote requests. For example, they can take the following measures:

Reviewer: ```suggestion
To solve this issue, merchants may need to react faster to quote requests. For example, they can take the following measures:
```

> * Clarify the essential details upfront with customers.

Reviewer: ```suggestion
* Clarify the essential details ahead of time with customers.
```

* Automate the RFQ process where possible: create assignment criteria and define workflows with predefined thresholds per customer if needed.

> * Notify customers about the processing of their RFQs through, for example, email with provided estimated delivery dates.

Reviewer: ```suggestion
* Notify customers about the processing of their RFQs, for example through an email with the provided estimated delivery dates.
```

> * Notify merchants about new or updated RFQ through, for example, email.

Reviewer: ```suggestion
* Notify merchants about new or updated RFQs through email or other means.
```

> ## Use Case 2: Marketplace customers must create separate carts and RFQs for each merchant

Reviewer 2: ```suggestion
## Use case 2: Marketplace customers must create separate carts and RFQs for each merchant
```

In Marketplace, if customers have several items from different merchants in their cart, they have to send an RFQ for each item separately. This means they have to create separate carts for products from different merchants they want to request quotes from. 

### Solution

> To solve this issue, Spryker needs to introduce Marketplace support for RFQ. To achieve this, the following actions are required:

Reviewer: ```suggestion
To solve this issue, Spryker needs to introduce Marketplace support for RFQ. To achieve this, the following actions are required:
```
Unsure of meaning, does this mean that the feature has not yet been introduced or that this can be done already?

Author: To my best knowledge, Marketplace does not support RFQ

Reviewer: Got it. I'm just unsure why we're advertising a feature for marketplace that's not there yet - to clarify this is just me being curious, wondering abt the reasoning no more!

* Introduce the Split Quote feature so that different carts and quotes per merchant are automatically created.

> * Display quote requests in Merchant Portal so that merchants can manage the requests (reply to the requests and change prices) or allow merchants to log in as an [agent assist](docs/scos/user/features/{{site.version}}/agent-assist-feature-overview.html) with the limitation of only seeing quote requests with their products or offers.

Reviewer: ```suggestion
* Display quote requests in Merchant Portal so that merchants can manage the requests, e.g. replying to the requests and changing prices, or allow merchants to log in as an [agent assist](docs/scos/user/features/{{site.version}}/agent-assist-feature-overview.html) with the limitation of only seeing quote requests for their products or offers.
```

---

> title: "Use cases: Request for Quote processes"

Author: title and file name must be identical. So we'll have to rename the file name too in this case.
@jeremyfourna I'm not sure I understand what you want to remove: the words "for quote" or "request for quote" or something else.

> title: "Use cases: Request for Quote processes"

Reviewer 3: I mean, remove the quotes ("") at the start and end of the title.

> title: "Use cases: Request for Quote processes"

Author: when we use a colon (:) in the title we have to use quotes. Otherwise, the site won't build

> title: "Use cases: Request for Quote processes"

Reviewer 3: OK but they wont be displayed right?

> title: "Use cases: Request for Quote processes"

Author: Right

description: This document describes the use cases for Requests for Quote.
last_updated: Nov 15, 2022
template: concept-topic-template
---

> This document describes the use cases of [Requests for Quote (RFQ)](/docs/pbc/all/request-for-quote/request-for-quote.html) and proposed solutions.

Author: ```suggestion
```

> This document describes the use cases of [Requests for Quote (RFQ)](/docs/pbc/all/request-for-quote/request-for-quote.html) and proposed solutions.

Author: done

> ## Use Case 1: Long waiting time until RFQ gets processed

Author: ```suggestion
## Use Case 1: Long waiting time until Request for Quote (RFQ) gets processed
```

> ## Use Case 1: Long waiting time until RFQ gets processed

Author: Fixed

> The request for quote process is currently not entirely efficient because it is manual. As a result, sometimes, customers have to wait for a long time until their RFQ gets approved or denied. 

Author: @jeremyfourna Provide please more details about what you would like to rephrase. For me, it looks okay, so I need more info to fix it.

> The request for quote process is currently not entirely efficient because it is manual. As a result, sometimes, customers have to wait for a long time until their RFQ gets approved or denied. 

Reviewer 3: Something like this
> If your customers have to wait for a long time until their RFQ gets approved or denied, you can take advantage of Spryker RFQ capability to automate this process.

> The request for quote process is currently not entirely efficient because it is manual. As a result, sometimes, customers have to wait for a long time until their RFQ gets approved or denied. 

Reviewer 3: OK then ignore my comment

### Solution

To solve this issue, merchants must react faster to quote requests. For example, they can take the following measures:
* Clarify the essential details upfront with customers.
* Automate the RFQ process where possible: create assignment criteria and define workflows with predefined thresholds per customer if needed.
* Notify customers about the processing of their RFQs through, for example, email with provided estimated delivery dates.
* Notify merchants about new or updated RFQ through, for example, email.

> ## Use Case 2: Marketplace customers must create separate carts and RFQs for each merchant.

Author: ```suggestion
## Use Case 2: Marketplace customers must create separate carts and RFQs for each merchant
```

In Marketplace, if customers have several items from different merchants in their cart, they have to send an RFQ for each item separately. This means they have to create separate carts for products from different merchants they want to request quotes from. 

### Solution

> To solve this issue, Spryker needs to introduce Marketplace support for RFQ. To achieve this, the following actoins are requried from the Spryker side:

Author: ```suggestion
To solve this issue, Spryker needs to introduce Marketplace support for RFQ. To achieve this, the following actions are required:
```
